training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb

```diff
@@ -859,7 +852,7 @@
     "smp_estimator = PyTorch(\n",
     "    entry_point=\"train.py\",\n",
     "    hyperparameters=hyperparameters,\n",
-    "    source_dir=os.getcwd(),\n",
+    "    source_dir=os.path.join(os.getcwd(), \"../shared-scripts\"),\n",
```

Comment:
Thanks for the change, I assume that you have tried this E2E.

Reply:
Yes, I tested on one notebook E2E which worked, so assuming will work for all the notebooks.